package.json

```diff
@@ -133,9 +135,9 @@
     }
   },
   "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx,json,css,md,scss}": [
-      "prettier --write",
-      "git add"
+    "*.{ts,tsx}": [
+      "eslint --fix",
+      "prettier --write"
```

Comment:
What lint -staged? Is this for CI? Do we need to write here?

Reply:
This is the pre-commit hook, which can fix some eslint issues and format issues automatically when you run `git commit`. So yeah, `--write` is needed to actually fix that file. (FYI the CI check runs `npm lint`, which runs `prettier --check` instead of `--write`.)